### ISSUE TITLE
Please add a mute feature

### ISSUE THREAD

**Issue by felixplesoianu** [jaywink/socialhome#205]
Sorry to pester again, but I literally just joined a few hours ago and someone, just one account, is swamping the public stream with endless ranting. Please make it possible to mute certain users. Thanks.

**Comment by jaywink** [comment]
You're certainly not pestering. Feedback is valuable, thank you for that 🙇 

Mute, or "hiding content from a user" is covered by #140 , just hasn't been implemented yet. I'll keep this open and hoping to implement this sooner than later. I agree the public stream is hard to follow otherwise if you can't hide profiles that are too verbose or just not interesting.

Thanks again for the feedback! 👍
